llama_stack/distribution/library_client.py

```diff
@@ -81,30 +81,40 @@ def convert_to_pydantic(annotation: Any, value: Any) -> Any:
         return value
 
     origin = get_origin(annotation)
+
     if origin is list:
         item_type = get_args(annotation)[0]
         try:
             return [convert_to_pydantic(item_type, item) for item in value]
         except Exception:
-            print(f"Error converting list {value}")
+            print(f"Error converting list {value} into {item_type}")
```

Comment:
not in this PR - but shouldnt we be logging errors differently than with prints?

Reply:
yeah, let me update in a separate PR along with other prints.

Reply:
removed prints in https://github.com/meta-llama/llama-stack/pull/1146